src/components/harvestLogs/HarvestLogDrawer.tsx

```diff
@@ -428,13 +447,67 @@ const HarvestLogDrawer = ({
           Cancel
         </Button>
 
-        <Button
-          variant="contained"
-          onClick={handleSubmit(onSubmit)}
-          disabled={isLoading}
-        >
+        <Button variant="contained" onClick={handleOpen} disabled={isLoading}>
           Save
         </Button>
+
+        <Dialog open={open} onClose={handleClose}>
+          <div
+            style={{
+              display: "flex",
+              flexDirection: "column",
+              alignItems: "center",
+              justifyContent: "center",
+              padding: "40px 24px 28px 24px",
+              borderRadius: "16px",
+            }}
+          >
+            <TelegramLogo size={40} style={{ color: "#055E40" }} />
+            <DialogTitle
+              style={{
+                fontSize: "30px",
+                fontStyle: "normal",
+                fontWeight: 600,
+                lineHeight: "38px",
+                letterSpacing: "-0.3px",
+                paddingTop: "20px",
+              }}
+            >
+              Harvest Entry Completed!
+            </DialogTitle>
+            <DialogContent
+              style={{
+                paddingBottom: "32px",
+                fontSize: "16px",
+                fontStyle: "normal",
+                fontWeight: 500,
+                lineHeight: "24px",
+              }}
+            >
+              {selectedPicker?.label} will receive an SMS with the summary.
+            </DialogContent>
+            <DialogActions
+              style={{ justifyContent: "space-between", width: "100%" }}
+            >
+              <Button
+                onClick={handleClose}
+                style={{
+                  border: "1px solid var(--Colors-Brand-500, #055E40)",
```

Comment:
the button should not be modified, if there is an issue with the button we have to add it to the theme config, otherwise, the design has to be fixed.